core-guides/core-data-curators.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Welcome, Data Curators
+---
+
+As mentioned in the introductory page, one key issue is that the problem with open data is not a matter of *creation*, rather *curation*. This means the role of a Data Curator is to take data from known (quality-proven and reliable) sources and 
+transform that into data packages.
+
+This guide will introduce you to key routines we usually do, before publishing any data package.
+
+# Getting data
+
+The first step is, naturally, finding data to work with. There are two common ways you can do this:
+
+1. Follow the [GitHub's issue tracker](https://github.com/datasets/registry/issue) and look for a package with 3-star priority, or
+
+2. Get your own data and start preparing it.
+
+Which one you should go to first is really up to you. The difference, however, is that some are more needed than others and 
+you could use that knowledge and skill to tackle on some of the project's priorities.
+
+Other than that, the workflow and requirements is pretty much the same.
+
+## Following urgencies
+
+The main advantage of working with a previously open issue is the fact you are introduced to some work that is already 
+developed. Chances are that someone has already took care of finding the best source, convert it to CSV or XLSX and then your 
+work is reduced significantly.
+
+## Your Own Taste
+
+You can also search for topics of your preference, or even suggest them in the issues page if they are not there already.
+
+
+# Preparing the Data
+
+Now, back to the workflow. This is the troublesome part. The job is to, first, find the best and most reliable source, convert 
+if to CSV or XLSX and then work on preparing the data package. If you do not know what `CSV` is, we advise you to read the 
```

Comment:
The audience I though for these guides would be non-power users. To me, I can simply download data from the web, whether it is excel, csv or an API. But for a regular user, perhaps things are not that straightforward. I can remove it if you think it is better ;)